testing/upload_validation.ts

```diff
@@ -91,6 +91,12 @@ import * as z from 'zod';
  * a real RegExp object.
  */
 export const PACKS_VALID_COLUMN_FORMAT_MATCHER_REGEX = /^\/(.*)\/([a-z]+)?$/;
+export const BUILDING_BLOCK_COUNT_LIMIT = 500;
```

Comment:
Nit, oof, I don't love that we were using deprecated style for constant variable names here, rather than being consistent with the unlovely style, mind just updating them all to use UpperCamelCase?